backend/package.json

```diff
@@ -26,20 +26,20 @@
     "@types/express": "^4.17.13",
     "@types/fs-extra": "^9.0.13",
     "@types/memoizee": "^0.4.8",
-    "@types/pacote": "^11.1.3",
-    "objection": "^2.2.18",
-    "ts-node-dev": "^1.1.8",
-    "typescript": "^4.6.4"
+    "@types/pacote": "^11.1.5",
+    "objection": "^3.0.1",
```

Comment:
I'd keep this at v2 for simplicity, only because we use Objection purely for types in viewer, not functionality. The new DB update will remove this necessity